src/server/api/lib/campaign.ts

```diff
@@ -274,6 +275,27 @@ export const copyCampaign = async (options: CopyCampaignOptions) => {
         [newCampaign.id, campaignId]
       );
 
+      // Copy Campaign Variables
+      await trx.raw(
+        `
+          insert into campaign_variable (campaign_id, display_order, name, value)
+          select
+            ? as campaign_id,
+            display_order,
+            campaign_variable.name,
+            (case
+              when all_campaign.is_template then null
```

Comment:
Suggestion: We should create an issue for disabling input of a campaign variable value on the template campaign builder page.